vendor/BUILD

```diff
@@ -90,6 +90,7 @@ go_library(
         "github.com/Azure/azure-sdk-for-go/arm/network/version.go",
         "github.com/Azure/azure-sdk-for-go/arm/network/virtualnetworkgatewayconnections.go",
         "github.com/Azure/azure-sdk-for-go/arm/network/virtualnetworkgateways.go",
+        "github.com/Azure/azure-sdk-for-go/arm/network/virtualnetworkpeerings.go",
```

Comment:
It's not used in this PR but it seems `hack/update-bazel.sh` wants that to be there.